.github/workflows/documentation.yml

```diff
@@ -1,24 +1,28 @@
+
 name: Generate Terraform README docs
 on:
   workflow_dispatch:
   pull_request:
     branches:
       - main
-      -
-permissions: read-all
+
+permissions:
+  contents: read
 
 jobs:
   docs:
+    permissions:
+      pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
```

Comment:
you are downgrading it from v3.5.0 to 3.4.0 here; could you leave it at 3.5.0 please?

Reply:
Sorry yes, bumped to 3.5.2 as per https://github.com/ministryofjustice/modernisation-platform-terraform-module-template/blob/main/.github/workflows/documentation.yml